_includes/pbc/all/install-features/202304.0/install-the-gift-cards-feature.md

```diff
@@ -0,0 +1,1338 @@
+
+**src/data/import/glossary.csv**
+
+```yaml
+cart.total.price_to_pay,Zu bezahlender Betrag,de_DE
+cart.total.price_to_pay,Price to pay,en_US
+cart.giftcard.label,Gift card,en_US
+cart.giftcard.label,Geschenkgutschein,de_DE
+cart.giftcard.apply.failed,The gift card is not applicable,en_US
+cart.giftcard.apply.failed,Geschenkgutschein konnte nicht angewendet werden,de_DE
+cart.giftcard.apply.successful,Your gift card code has been applied,en_US
+cart.giftcard.apply.successful,Ihr Geschenkgutschein wurde angewendet,de_DE
+cart.voucher.apply.successful,Your voucher code has been applied,en_US
+cart.voucher.apply.successful,Ihr Gutscheincode wurde angewendet,de_DE
+cart.code.apply.failed,Code could not be applied,en_US
+cart.code.apply.failed,Gutscheincode konnte nicht angewendet werden,de_DE
+general.next.button,Next,en_US
+general.next.button,Weiter,de_DE
+checkout.giftcard.label,Gift card,en_US
+checkout.giftcard.label,Geschenkgutschein,de_DE
+roduct.attribute.value,Value,en_US
+product.attribute.value,Der Wert,de_DE
+mail.giftCard.delivery.subject,Your Gift card!,en_US
+mail.giftCard.delivery.subject,Deine Geschenkkarte!,de_DE
+mail.giftCard.delivery.text,"Sehr geehrter Kunde, vielen Dank für den Kauf einer Geschenkgutschein in unserem Shop. Ihr Gutscheincode lautet: ",de_DE
+mail.giftCard.delivery.text,"Dear customer, thank you for buying a gift card at our shop. Your gift card code is: ",en_US
+mail.giftCard.usage.subject,Thank you for using a Gift Card!,en_US
+mail.giftCard.usage.subject,Vielen Dank dass Sie ein Geschenkgutschein benutzt haben.,de_DE
+cart.code.enter-code,Gutscheincode/Geschenkgutscheincode eingeben,de_DE
+cart.code.enter-code,Enter voucher/gift card code,en_US
+```
+
+2. Apply glossary keys:
+
+```bash
+console data:import:glossary
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that in the database, the configured data has been added to the `spy_glossary` table.
+
+{% endinfo_block %}
+
+### 3) Set up widgets
+
+Register the following global widget:
+
+| WIDGET | SPECIFICATION | NAMESPACE |
+| --- | --- | --- |
+| CartCodeFormWidget |Provides a cart code activation form.  | SprykerShop\Yves\CartCodeWidget\Widget |
+
+**src/Pyz/Yves/ShopApplication/ShopApplicationDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\ShopApplication;
+
+use SprykerShop\Yves\CartCodeWidget\Widget\CartCodeFormWidget;
+use SprykerShop\Yves\ShopApplication\ShopApplicationDependencyProvider as SprykerShopApplicationDependencyProvider;
+
+class ShopApplicationDependencyProvider extends SprykerShopApplicationDependencyProvider
+{
+    /**
+     * @return string[]
+     */
+    protected function getGlobalWidgets(): array
+    {
+        return [
+            CartCodeFormWidget::class,
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the widget is displayed on the **Cart** page and the **Summary** page of the checkout process.
+
+{% endinfo_block %}
+
+### 4) Enable controllers
```

Comment:
If this section have only one sub step, we can merge them or replace the step name with the sub-step name as follows:
```suggestion
### 4) Enable the route list
```